### ISSUE TITLE
Build runtime=release-static variant by default on Windows

### ISSUE THREAD

**Issue by OlafvdSpek** [boostorg/build#79]
Could the variant with static release runtime be build by default?

It's a frequently used option to allow xcopy deployments.

**Comment by vprus** [comment]
Is there a demonstrated consensus over at mailing lists to build that one by default?

**Comment by OlafvdSpek** [comment]
No(t yet). Is one necessary?

**Comment by vprus** [comment]
I'm afraid so. Adding another variable to the matrix has significant cost for every user, so I'd rather not do that unless we're sure that a significant percentage will be happy.

**Comment by OlafvdSpek** [comment]
Right. Perhaps a note could be added to the getting started guide then.

**Comment by vprus** [comment]
Yeah, though it appears that getting started guide no longer builds - with a problem somewhere between docutils and Python. Will tweak the wording when I get past that hurdle.
